web/main.ts

```diff
@@ -15,10 +17,24 @@ import twindConfig from "./twind.config.ts";
 import { WebUIUrlEnvKey } from "../constants.ts";
 import { combinedEnv } from "../runHelpers.ts";
 
+import { PrismaClient } from "../generated/client/deno/edge.ts";
+
+export const prisma = new PrismaClient({
+  datasources: {
+    db: {
+      url: env.DATABASE_URL,
+    },
+  },
+});
+
+if (!env.APP_KEY) Deno.env.set("APP_KEY", crypto.randomUUID());
```

Comment:
sadly fresh-session cookie store seems to use hardcoded envvar name..
https://github.com/Octo8080X/fresh-session/blob/main/src/stores/cookie.ts#L14
maybe we can use kv or redis store instead of cookie store or use some other name re-exposing as APP_KEY